text/0000-alternate-registries.md

```diff
@@ -102,6 +107,12 @@ plus has the ability to have crates pushed to it, however this has the following
 * It requires crates.io to be able to combine two registries, or requires a radical change to the way crates.io works
 * The current proposal could be extended to support this, if a caching server is added at a later stage
 
+## Including registry definitions in a global location
+We considered using a global configuration file (eg ~/.cargo/config) to allow a registry to
```

Comment:
Sorry, that was a reflection of discussions that I had within our company prior to submitting the RFC, not intended as a full reflection of the discussion here as I thought it would be useful to include.